Customize the greeting message.

Copyright © 2024 Red Hat, Inc.
Copyright © 2024 林博仁(Buo-ren Lin) <buo.ren.lin@gmail.com>
SPDX-License-Identifier: CC-BY-SA-3.0

--- cello.c.orig	2024-12-14 18:17:30.207567293 +0800
+++ cello.c	2024-12-15 16:32:33.392437058 +0800
@@ -8,6 +8,6 @@
 #include <stdio.h>
 
 int main(void) {
-    printf("Hello World\n");
+    printf("Hello World from my very first patch!\n");
     return 0;
 }
